dotCMS/src/main/java/com/dotmarketing/util/PageRequestModeUtil.java

```diff
@@ -4,28 +4,24 @@
 import javax.servlet.http.HttpSession;
 
 /**
- * Created by freddyrodriguez on 5/2/16.
+ * 
+ * Use {link {@link PageMode}
+ * @author will
+ *
  */
+@Deprecated
 public abstract  class PageRequestModeUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)